lib/node_modules/@stdlib/stats/base/dists/rayleigh/cdf/benchmark/benchmark.js

```diff
@@ -64,17 +64,22 @@ bench( pkg, function benchmark( b ) {
 bench( pkg+':factory', function benchmark( b ) {
 	var sigma;
 	var mycdf;
+	var len;
 	var x;
 	var y;
 	var i;
 
 	sigma = 4.0;
 	mycdf = cdf.factory( sigma );
-
+	len = 100;
+	x = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -25.0, 50.0 );
```

Comment:
Let's keep the original range:

```suggestion
		x[ i ] = uniform( -25.0, 25.0 );
```